Resources/public/js/views/ez-languageselectionboxview.js

```diff
@@ -354,13 +379,13 @@ YUI.add('ez-languageselectionboxview', function (Y) {
             },
 
             /**
-             * Already existing translations of content
+             * List of languages used as reference
```

Comment:
here you should mention that in translation mode, this is the list of existing translations while not in translation mode, this is the list languages the user can pick in.